backends-common/jpa/pom.xml

```diff
@@ -15,20 +15,20 @@
             <artifactId>guava</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.apache.openjpa</groupId>
-            <artifactId>openjpa</artifactId>
-            <version>2.4.2</version>
+            <groupId>com.h2database</groupId>
```

Comment:
No, IMO dependencies on other apache project should be treated as external dependencies (IE we don't control them at all, it is not part of this project).

At least this tool will give us a good, common definition of ordering POM.